doc/source/howtos/virt_sandbox.rst

```diff
@@ -117,12 +41,13 @@ scales well with application servers.
 
       Do not use default credentials in production!
 
-   The following example shows how to connect to the ProxySQL admin interface
-   with default credentials:
+   The following example shows how to connect to the |proxysql| admin interface
```

Comment:
Since we are connected to ProxySQL node we can say `(assuming that |proxysql| IP is 192.168.70.74):`